vzdqload.c: Define __USE_ISOC99 properly to fix build with fresh toolchain

--- vzquota-3.0.12/src/vzdqload.c
+++ vzquota-3.0.12/src/vzdqload.c
@@ -20,7 +20,9 @@
 #include <sys/types.h>
 
 /* to include vsscanf() */
-#define __USE_ISOC99
+#ifndef __USE_ISOC99
+# define __USE_ISOC99
+#endif
 #include <stdio.h>
 
 #include <stdlib.h>
